spec/controllers/api/api_tags_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
```

Comment:
@simonv3 Can you explain this comment? Google and stackoverflow only told me about ways to disable this feature. I'm not sure how to fix this.